src/components/sbb-tag-group/sbb-tag-group.tsx

```diff
@@ -0,0 +1,64 @@
+import { Component, ComponentInterface, Element, h, JSX, Listen } from '@stencil/core';
+import { getNextElementIndex, isArrowKeyPressed } from '../../global/helpers/arrow-navigation';
+import { hostContext } from '../../global/helpers/host-context';
+
+/**
+ * @slot unnamed - Provide one or more 'sbb-tag' to add to the group.
+ */
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-tag-group.scss',
+  tag: 'sbb-tag-group',
+})
+export class SbbTagGroup implements ComponentInterface {
+  @Element() private _element: HTMLElement;
+
+  private _isNested: boolean;
+
+  public tags: HTMLSbbTagElement[];
+
+  private _getTags(): HTMLSbbTagElement[] {
+    return Array.from(this._element.querySelectorAll('sbb-tag')) as HTMLSbbTagElement[];
+  }
+
+  private _onTagSlotChange(): void {
+    this.tags = this._getTags();
+  }
+
+  public connectedCallback(): void {
+    this._isNested = !!hostContext('sbb-tag-group', this._element);
+  }
+
+  @Listen('keydown')
+  public handleKeyDown(evt: KeyboardEvent): void {
+    const enabledTags: HTMLSbbTagElement[] = this._getTags().filter(
+      (tag) => !tag.hasAttribute('disabled')
+    );
+
+    if (
+      !enabledTags ||
+      // don't trap nested handling
+      ((evt.target as HTMLElement) !== this._element &&
+        (evt.target as HTMLElement).parentElement !== this._element)
+    ) {
+      return;
+    }
+
+    if (isArrowKeyPressed(evt)) {
+      const cur: number = enabledTags.findIndex((e: HTMLSbbTagElement) => e === evt.target);
+      const nextIndex: number = getNextElementIndex(evt, cur, enabledTags.length);
+      enabledTags[nextIndex]?.focus();
+    }
+  }
+
+  public render(): JSX.Element {
+    if (this._isNested) {
+      throw new Error('Error: nested sbb-tag-group.');
```

Comment:
Is there a reason we need to catch this?

Reply:
Taken from [here](https://github.com/lyne-design-system/lyne-components/issues/1467): "not consider nesting of groups (no sbb-tag-group inside another sbb-tag-group)".

Shoud I just return? or render some kind of error like [here](https://github.com/lyne-design-system/lyne-components/blob/master/src/components/sbb-card-product/sbb-card-product.tsx#L214)?

Reply:
I think we shouldn't do anything, it was just meant, that we don't have to support nested tag groups